docs/docs/debug/csrf.mdx

```diff
@@ -40,13 +40,17 @@ Before starting to debug cookie and CSRF issues, make sure to check out the
 Chrome Developer Tools (or any comparable technology) Cookies tabs in the
```

Comment:
Please ensure that `csrf.md` is not referenced anywhere

Reply:
Did not find it referenced in the `docs` directory anywhere.